sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjb.java

```diff
@@ -357,6 +357,7 @@ public EventParticipantDto saveEventParticipant(@Valid EventParticipantDto dto,
 			// The Event Participant is newly created, let's check if the related person is related to other events
 			// In that case, let's notify corresponding responsible Users of this relation
 			notifyEventResponsibleUsersOfCommonEventParticipant(entity, event);
+			entity.setArchived(event.isArchived());
```

Comment:
The change should be done before `service.ensurePersisted`, maybe within `fillOrBuildEntity` or directly afterwards.